test/common/schemaform/definitions/address.unit.spec.jsx

```diff
@@ -17,152 +16,127 @@ describe('Schemaform definition address', () => {
     //  fields but this doesn't work.
     // const requiredInputs = formDOM.querySelectorAll('input[required=true]');
-    const requiredInputs = formDOM.querySelectorAll('label span.schemaform-required-span');
+    const requiredInputs = form.find('label').find('span.schemaform-required-span');
     expect(requiredInputs.length).to.not.equal(0);
   }).timeout(4000);
 
   it('should update labels and state selection conditionally', () => {
     const s = schema(addressSchema, false);
     const uis = uiSchema();
-    const form = ReactTestUtils.renderIntoDocument(
+    const form = mount(
       <DefinitionTester
         schema={s}
         uiSchema={uis}/>
     );
 
-    const formDOM = findDOMNode(form);
-    const find = formDOM.querySelector.bind(formDOM);
-
-    // By default, the Country is USA, so the postalCode label should be 'ZIP Code'
-    //  and the State label should be 'State' and the field should be a dropdown
-    const labels = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'label');
-    const postalCodeLabel = labels.find(label => label.htmlFor === 'root_postalCode');
-    const stateLabel = labels.find(label => label.htmlFor === 'root_state');
-    const stateField = formDOM.querySelector('#root_state');
+    const labels = form.find('label');
+    const postalCodeLabel = labels.findWhere(label => label.props().htmlFor === 'root_postalCode');
+    const stateLabel = labels.findWhere(label => label.props().htmlFor === 'root_state');
+    const stateField = form.find('select#root_state');
 
     // Check the labels' text
-    expect(postalCodeLabel.textContent === 'ZIP Code');
-    expect(stateLabel.textContent === 'State');
+    expect(postalCodeLabel.text()).to.equal('Postal code');
+    expect(stateLabel.text()).to.equal('State');
 
     // And state input type / options
-    expect(stateField.tagName === 'select');
-    // jsdom doesn't support namedItem; use options instead
-    expect(Array.from(stateField.options).find(op => op.value === 'OR')).to.not.be.undefined;
+    expect(stateField.find('option').someWhere(n => n.props().value === 'OR')).to.be.true;
 
     // Entering a military city should result in different "state" options
-    const cityField = formDOM.querySelector('#root_city');
-    ReactTestUtils.Simulate.change(cityField, {
-      target: {
-        value: 'apo'
-      }
-    });
-    expect(Array.from(stateField.options).find(op => op.value === 'AA')).to.not.be.undefined;
+    fillData(form, 'input#root_city', 'apo');
+    expect(stateField.find('option').someWhere(n => n.props().value === 'AA')).to.be.true;
 
     // Change the country
-    const countryField = find('#root_country');
-
-    ReactTestUtils.Simulate.change(countryField, {
-      target: {
-        value: 'CAN'
-      }
-    });
+    fillData(form, 'select#root_country', 'CAN');
 
     // Check to see if the postal code and state updated
-    expect(stateLabel.textContent === 'Provice');
```

Comment:
wonder how this ever passed

Reply:
it just resolves to expect(false), which isn't an assertion. I'm not sure how we missed that originally.